pkg/applier/applier.go

```diff
@@ -179,7 +179,8 @@ func (a *Applier) UpdateInPath(
 	return a.Updates(us)
 }
 
-// Deprecated: Please use another CreateOrUpdate methods with a YamlStringReader
+// Deprecated: Please use another CreateOrUpdateInPath or CreateOrUpdateResouces
+// methods with a YamlStringReader
```

Comment:
deprecated? because the data is not from the reader?

Reply:
didn't notice it before 😂

Reply:
Do you use it?